src/Silex/Provider/HttpFragmentServiceProvider.php

```diff
@@ -97,4 +96,25 @@ public function boot(Application $app)
     {
         $app['dispatcher']->addSubscriber($app['fragment.listener']);
     }
+
+    /**
+     * @return string
+     */
+    protected function getUriSignerSecret(): string
+    {
+        if ($this->uriSignerSecret) {
+            return $this->uriSignerSecret;
+        }
+
+        $this->uriSignerSecret = getenv('SPRYKER_ZED_REQUEST_TOKEN') ?: null;
```

Comment:
This env variable is a part of Spryker, and cannot be used in SIlex lib.
Also, a fix for this module was already provided.
Please, read the expected behavior from the ticket carefully.